src/gui/uithememanager.h

```diff
@@ -54,6 +55,7 @@ class UIThemeManager : public QObject
     QString getIconPath(const QString &iconId) const;
 
     static UIThemeManager *m_instance;
+    class ApplicationPalette *m_applicationPalette; // stylesheet would not be examined till some widget's show event so we need to keep it around
```

Comment:
the `class` word is redundant.

Reply:
done. Foreward declared it before class def.